lesson-003-web-server-graceful-shutdown/README.md

```diff
@@ -0,0 +1,16 @@
+
+## Trigger requests from the terminal with:
+```
+for ((i=1;i<=10;i++)); do curl http://localhost:8080;done
+```
+
+## commands to stop the server
+```
+pgrep -i main
+kill -SIGTERM 69489
```

Comment:
Some explanation here may be useful.
Also,it may be interesting to showcase Go code that will do this.

Reply:
Sure! The plan was to have an initial discussion around the code, and then build the README around it 👍